src/Open3D/GUI/SceneWidget.cpp

```diff
@@ -36,9 +36,7 @@
 
 #include <Eigen/Geometry>
 
-#include <iostream>
-
-#define ENABLE_PAN 0
+#define ENABLE_PAN 1
```

Comment:
Why do we need `#define` here? Do we need to turn off camera pan on some platforms?

Reply:
Originally it didn't entirely work. But I'm not convinced that we are going to like it, either. If you pan and then rotate, the rotate rotates about a different point (obviously) but I think it's going to be easy to get confused.